### PR TITLE
Add containerd and CRI-O to glossary

### PR DESCRIPTION
**Pull request by sftim**
2 new glossary entries:
 - containerd
 - CRI-O

Relevant to #5993. Could help with a [suggestion](https://github.com/kubernetes/website/pull/14255#discussion_r283097009) I made for PR #14255.

### CODE REVIEW

**Comment by sftim** [comment]
NB _containerd_ sorts after the initial-caps entries due to #7491

File: content/en/docs/reference/glossary/containerd.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: containerd
+id: containerd
+date: 2019-05-14
+full_link: https://containerd.io/docs/
+short_description: >
+  Container runtime with an emphasis on simplicity, robustness and portability
```
Comment (tengqm) [inline content/en/docs/reference/glossary/containerd.md]:
```suggestion
  A container runtime with an emphasis on simplicity, robustness and portability
```

File: content/en/docs/reference/glossary/cri-o.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: CRI-O
+id: cri-o
+date: 2019-05-14
+full_link: https://cri-o.io/docs/
+short_description: >
+  Lightweight container runtime specifically for Kubernetes
```
Comment (tengqm) [inline content/en/docs/reference/glossary/cri-o.md]:
```suggestion
  A lightweight container runtime specifically for Kubernetes
```

File: content/en/docs/reference/glossary/cri-o.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+aka:
+tags:
+- tool
+---
+A container runtime that lets OCI-compliant runtimes work with Kubernetes.
```
Comment (tengqm) [inline content/en/docs/reference/glossary/cri-o.md]: Reword this sentence?

Comment (sftim) [reply content/en/docs/reference/glossary/cri-o.md]: Reworded!

**Pull request merged**